content/learn/javascript-callbacks.en.md

```diff
@@ -125,7 +125,7 @@ window.addEventListener('load', () => {
 });
 ```
 
-This is just a simple 4-levels code, but I've seen much more levels of nesting and it's not fun.
+This is just a simple 4-levels code, but we've seen much more levels of nesting and it's not fun.
```

Comment:
```suggestion
This is just a simple 4-levels code, but there have been much more levels of nesting and it's not fun.
```

How about using "there've been"?

Reply:
It doesn't depict that the authors have actually faced/seen the mentioned scenario. My first question after reading that is 'Where have been? It is little unclear, whereas, i think, "we've seen" is actually more strongly presenting the authors argument.